api-server-rest/openapi/api.json

```diff
@@ -1,5 +1,5 @@
 {
-  "openapi": "3.0.3",
+  "openapi": "3.1.0",
```

Comment:
I guess this is unrelated to this PR?

Reply:
Yes. It updates the openapi automatically generated file in a separate commit. if needed i can move it to another pr

Reply:
no, it's fine. I noticed the same, locally.